src/sources/socket/udp.rs

```diff
@@ -199,14 +218,46 @@ pub(super) fn udp(
 
                                 for event in &mut events {
                                     if let Event::Log(ref mut log) = event {
-                                        log.try_insert(log_schema().source_type_key(), Bytes::from("socket"));
-                                        log.try_insert(log_schema().timestamp_key(), now);
-                                        log.try_insert(host_key.as_str(), address.ip().to_string());
+                    log_namespace.insert_vector_metadata(
```

Comment:
Weirdly, this is what rustfmt outputs :/

I could force it into the right formatting if folks feel strongly about it.